src/configure/configure.ts

```diff
@@ -171,10 +174,13 @@ class Orchestrator {
                 else {
                     this.context['rightClickScenario'] = true;
                 }
+
                 if (this.inputs.targetResource.resource && this.inputs.targetResource.resource.id) {
                     this.context['resourceId'] = this.inputs.targetResource.resource.id;
                 }
                 this.selectTemplate(this.inputs.targetResource.resource);
+                telemetryHelper.setTelemetry(TelemetryKeys.SelectedTemplate, this.inputs.pipelineConfiguration.template.label);
```

Comment:
would it be better to log template Id instead of label?

Reply:
The local templates don't have template id.